homepage/__init__.py

```diff
@@ -0,0 +1,6 @@
+import check50
+
+@check50.check()
+def submitted():
+    """Homepage submitted"""
+   
```

Comment:
Can't have an empty function body. Probably needs `pass`.

Reply:
I copied that from 2019/x  so assumed it was still the same.  Yes, @kzidane please review.

Reply:
`pass` isn't needed here because the docstring in Python is actual code that also happens to be used as a comment.

Reply:
🤯

Reply:
okay, I merged.